kroxylicious/pom.xml

```diff
@@ -332,10 +332,31 @@
         </profile>
         <profile>
             <id>debug</id>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.logging.log4j</groupId>
+                        <artifactId>log4j-bom</artifactId>
+                        <version>2.20.0</version>
```

Comment:
Please extract property.

Reply:
Out of interest why? We already have a place defining all the versions of the log4j dependencies (here), we don't really want people to add additional ones with manually managed versions.

Looking again i didn't use import scope here so need to fix it anyway

Reply:
Oh no I did just the diff view on my phone being poor